packages/view3d/src/core/Renderer.ts

```diff
@@ -221,7 +223,12 @@ class Renderer {
     camera.updatePosition();
 
     scene.shadowPlane.render();
-    threeRenderer.render(scene.root, camera.threeCamera);
+
+    if (postProcessing.isPostProcessing) {
+      this._view3D.postProcessing.composer.render();
+    }else {
+      threeRenderer.render(scene.root, camera.threeCamera);
```

Comment:
As `postProcessing` also has a render pass inside it, what about creating a class that composes a render pass and calling its render() only?

like,
```diff
- if (postProcessing.isPostProcessing) {
-   this._view3D.postProcessing.composer.render();
- } else {
-   threeRenderer.render(scene.root, camera.threeCamera);
- }
+ renderComposer.render();
```

Reply:
Check it!